benchmarks/apps/arepo/arepo.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 University College London (UCL) Research Software Development
+# Group.  See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+import os.path as path
+
+@rfm.simple_test
+class ArepoBenchmark(SpackTest):
+    valid_systems = ['-gpu']
+    valid_prog_environs = ['default']
+    spack_spec = 'arepo@master'  
```

Comment:
Building from the head of a branch is not recommended because it breaks reproducibility. There is no way to know later which commit of master was built. This is mentioned in [the documentation](https://ukri-excalibur.github.io/excalibur-tests/contributing/#spack-package). I'd normally recommend building off a tag or a release, but the repository does not seem to have any :facepalm: An alternative is to fetch a specific git commit. They describe how to do it in the [spack tutorial](https://spack.readthedocs.io/en/latest/packaging_guide.html#git)

Reply:
I was confused at the beginning not to see any tags and version for Arepo and now it makes sense! Thank you.